### PR TITLE
`number_of_leap_years` and `main` first PR

### PR DESCRIPTION
**Pull request by JWWJGit**
docs: finished first main and number_of_leap_years function

### CODE REVIEW

File: Calendar_CLI/Main.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from Calendar import number_of_days, number_of_leap_years, get_week_of_day
+from Log import log
```
Comment (YLee2121) [inline Calendar_CLI/Main.py]: In the design doc, it's called `write_log`.

File: Calendar_CLI/Main.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    """
+    Connect the other two modules and make this project work.
+    """
+    username = input("Welcome! Please tell us your name: ")
+    function_name = input("Which tool you want to use today? (day, week, year)")
+
+    if function_name == "day":
+        year = input("Year: ")
+        month = input("Month: ")
+        number_of_days(year, month)
```
Comment (YLee2121) [inline Calendar_CLI/Main.py]: you need to print out the return from the function

File: Calendar_CLI/Main.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        log(username, function_name)
+    elif function_name == "week":
+        year = input("Year: ")
+        month = input("Month: ")
+        day = input("Day: ")
+        number_of_days(year, month, day)
```
Comment (YLee2121) [inline Calendar_CLI/Main.py]: you need to print out the return from the function

File: Calendar_CLI/Main.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        log(username, function_name)
+    elif function_name == "year":
+        year1 = input("Year1: ")
+        year2 = input("Year2: ")
+        number_of_leap_years(year1, year2)
```
Comment (YLee2121) [inline Calendar_CLI/Main.py]: you need to print out the return from the function

File: Calendar_CLI/Main.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
+from Calendar import number_of_days, number_of_leap_years, get_week_of_day
+from Log import log
+
+
+def main() -> int():
```
Comment (YLee2121) [inline Calendar_CLI/Main.py]: for `main()`, no need to cite the return type. 
And also the `main()` doesn't return any integer.

File: Calendar_CLI/Calendar.py [hunk 6]
```diff
@@ -0,0 +1,9 @@
+def number_of_leap_years(year1: int, year2: int) -> int():
+    """
+    Return the number of leap years between year1, and year2, including year1 and year2 if there are also leap years.
```
Comment (YLee2121) [inline Calendar_CLI/Calendar.py:14]: follow the documentation guideline

image

**Review by YLee2121 — CHANGES_REQUESTED** [review]
some feedback

**Comment by JWWJGit** [comment]
Got it, thanks!

File: Calendar_CLI/Main.py [hunk 7]
```diff
@@ -0,0 +1,29 @@
+from Calendar import number_of_days, number_of_leap_years, get_week_of_day
```
Comment (YLee2121) [inline Calendar_CLI/Main.py]:
```python 
from Calendar_CLI.Calendar import number_of_days, number_of_leap_years, get_week_of_day
```

File: Calendar_CLI/Main.py [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+from Log import write_log
```
Comment (YLee2121) [inline Calendar_CLI/Main.py]:
```python
from Calendar_CLI.Log import write_log
```

File: Calendar_CLI/Main.py (hunk 7 above)
Comment (YLee2121) [reply Calendar_CLI/Main.py]: @Weirdojord

File: Calendar_CLI/Main.py (hunk 8 above)
Comment (YLee2121) [reply Calendar_CLI/Main.py]: @Weirdojord

**Comment by YLee2121** [comment]
It says you still have conflicts

**Review by YLee2121 — APPROVED** [review]
Approve

**Pull request merged**